### ISSUE TITLE
Cannot get emulation.js to work even with the example widgets

### ISSUE THREAD

**Issue by enaloha** [Matchstic/Xen-HTML#267]
**Describe the bug**
So I am trying to convert the Classic LS widget into XenHTML 2 (maybe someone has already done this?). I cannot get the emulation.js to work on my Mac, not even in the samples provided (e.g. Weather | Card). 

**To Reproduce**
1. Place Weather | Card folder on desktop
2. Copy emulation.js into Weather | Card folder
3. Edit emulation.js by including  <script src="emulation.js"></script> into <head> section
4. the html preview will still contain the variables and not the returned values

**Expected behavior**
See the actual values returned by the emulation.js script.

**Screenshots**
N/A

**Smartphone (please complete the following information):**
 - Device: iPhone 11 pro max
 - OS: iOS 13.5

**Additional context**
I am not the most proficient html/js coder, so maybe this is a really trivial mistake I am making.

**Comment by Matchstic** [comment]
Right now, emulation only support callback-based sytnax - inline support is not available just yet

**Comment by enaloha** [comment]
Thank you for the quick reply and clarification. May I ask whether there is any way to trigger events? I tried onclick, and it works on the desktop, but not in a lockscreen widget. Is touch input not supported or is there a trick to make it work?

**Comment by Matchstic** [comment]
onclick is the expected thing to be using. A working example for e.g. music buttons via onclick can be found in the default "Music | Player" widget.

**Comment by enaloha** [comment]
Interesting. Yes, the player does respond to onclick. For illustration purposes, here's a simple html  code that does not seem to work for me on the device, but works fine on a browser:

`<body>
<div id="info"><p id="text" onclick="myFunction()">Original Text</p></div>
<script>
function myFunction() {
	document.getElementById("text").innerHTML = "New Text";
}
</script>
</body>
`

**Comment by Matchstic** [comment]
That should work; make sure to check it in desktop Safari

**Comment by enaloha** [comment]
Yeah, it does. But if I put that into a widget, it won't when I tap on the text.
